x-pack/plugins/cross_cluster_replication/public/app/services/routing.js

```diff
@@ -84,8 +84,10 @@ class Routing {
     return encodeURI(`#${BASE_PATH}/auto_follow_patterns${section}/${encodeURIComponent(name)}`);
   };
 
-  getFollowerIndexPath = (name, section = '/edit') => {
-    return encodeURI(`#${BASE_PATH}/follower_indices${section}/${encodeURIComponent(name)}`);
+  getFollowerIndexPath = (name, section = '/edit', withBase = true) => {
```

Comment:
How about rewriting this to be `goToEditFollowerIndexPath` to make it a bit more convenient?

```js
goToEditFollowerIndex = (name) => {
  const uri = encodeURI(`#${BASE_PATH}/follower_indices/edit/${encodeURIComponent(name)}`);
  this.navigate(uri);
};
```

Reply:
just returning a string is more flexible for using the uri in other contexts, like `href`